### PR TITLE
DOCS: update README

### PR DESCRIPTION
**Pull request by corentinbettiol**
## Description

Add a "migrate from cmsplugin" management command mention in README.

## Related resources

image

## Checklist

* [x] I have opened this pull request against ``master``
* [ ] I have added or modified the tests when changing logic
* [ ] I have followed [the conventional commits guidelines](https://www.conventionalcommits.org/) to add meaningful information into the changelog
* [x] I have read the [contribution guidelines ](https://github.com/django-cms/django-cms/blob/develop/CONTRIBUTING.rst) and I have joined #workgroup-pr-review on 
[Slack](https://www.django-cms.org/slack) to find a “pr review buddy” who is going to review my pull request.

### CODE REVIEW

File: README.rst [hunk 1]
```diff
@@ -152,3 +152,15 @@ You can run tests by executing::
     :target: https://www.djangoproject.com/
 .. |djangocms| image:: https://img.shields.io/badge/django%20CMS-3.7%2B-blue.svg
     :target: https://www.django-cms.org/
+
+
+Updating from `cmsplugin-filer <https://github.com/django-cms/cmsplugin-filer>`_
+--------------------------------------------------------------------------------
+
+`cmsplugin-filer` was used a few times ago to create file, folder, image, link, teaser & video plugins on your DjangoCMS projects. Now the project is archived, and you might be wondering how to migrate your old instances without having to copy them manually to the new `djangocms-<file|picture|link|...>` plugins.
```
Comment (fsbraun) [inline README.rst:160]:
```suggestion
Historically, `cmsplugin-filer` was used to create file, folder, image, link, teaser & video plugins. Now `cmsplugin-filer` is archived, you can still migrate your old instances without having to copy them manually to the new `djangocms-<file|picture|link|...>` plugins.
```

File: README.rst [hunk 2]
```diff
@@ -152,3 +152,15 @@ You can run tests by executing::
...
+
+Don't be afraid, we have you covered! Here's a management command that you can use to get started with this tasks:
```
Comment (fsbraun) [inline README.rst:162]:
```suggestion
There's a third-party management command that supports your migration:
```

File: README.rst [hunk 3]
```diff
@@ -152,3 +152,15 @@ You can run tests by executing::
...
+
+`migrate_cmsplugin_filer.py <https://gist.github.com/corentinbettiol/84a6ea7e4d047fc01861b0af15fd60f0>`_
+
+This management command is only a starting point. It *may* work out of the box for some people, but we encourage you to read the code, understand what it does, and try it on a development environment before running it on your production server. The management command is only configured to transfer your `cmsplugin_link`, `cmsplugin_file`, `cmsplugin_folder` and `cmsplugin_image` plugins to modern `djangocms_*` plugins. If you need to transfer other `cmsplugin_*` plugins, you'll have to write your own code.
```
Comment (fsbraun) [inline README.rst:166]:
```suggestion
This management command is only a starting point. It *has* worked out of the box for some people, but we encourage you to read the code, understand what it does, and test it on a development environment before running it on your production server. 

The management command is configured to transfer your `cmsplugin_link`, `cmsplugin_file`, `cmsplugin_folder` and `cmsplugin_image` plugins to modern `djangocms_*` plugins. If you need to transfer other `cmsplugin_*` plugins, you'll have to write your own code.
```

**Review by fsbraun — COMMENTED** [review]
Hi @corentinbettiol ! Thank you for following up on this. Excellent. I only have some wording suggestions to support what you have written: This is a third-party starting point. Would you be ok with those.

File: README.rst (hunk 1 above)
Comment (corentinbettiol) [reply README.rst]: I fixed the content, thanks!

**Review by fsbraun — APPROVED** [review]

**Pull request merged**